web/src/app/browser/src/keyboardInterface.ts

```diff
@@ -20,6 +14,11 @@ export default class KeyboardInterface extends KeyboardInterfaceBase<ContextMana
 
     // Nothing else to do here... quite yet.  Things may not stay that way, though.
   }
+
+  public get osk() {
+    return this.engine.osk;
+  }
```

Comment:
What uses this?

Reply:
https://github.com/keymanapp/keyman/blob/0d43570e18070a8f418ecaff303d420e949831b4/web/src/app/browser/src/keyboardInterface.ts#L45-L81

These are what the Japanese CJK keyboard was falling over on.

Reply:
So couldn't we fixup `hideHelp`, `showHelp`, and `showPinnedHelp` to reference `this.engine.osk` instead?